### PR TITLE
revert use of randomUUID

### PR DESCRIPTION
**Pull request by edewit**
fixes: #4442

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

**Comment by SamTV12345** [comment]
Is [this](https://www.npmjs.com/package/uuid) library not a better replacement for the problem? We currently hope that Math.random()*1000 doesn't generate the same number. UUIDs are [specified](https://www.rfc-editor.org/rfc/rfc4122) and should never appear twice.

**Comment by edewit** [comment]
@SamTV12345 the number is used to identify an alert that stays on the page for 8 seconds so at most you have 3 of those at the same time. Even if they have the same number by change the worst that is going to happen is that one disappears a bit quicker. I don't think it's worth adding an entire library to prevent that.

**Comment by SamTV12345** [comment]
@edewit It is just a bit cleaner with UUIDs. But I guess it won't make a difference in practice. I created another pr that solves the problem with the library included.

File: apps/admin-ui/src/components/dynamic/MapComponent.tsx [hunk 1]
```diff
@@ -18,8 +18,9 @@ import type { ComponentProps } from "./components";
 import { convertToName } from "./DynamicComponents";
 
 type IdKeyValueType = KeyValueType & {
-  id: string;
+  id: number;
 };
+const generateId = () => Math.floor(Math.random() * 100);
```
Comment (jonkoops) [inline apps/admin-ui/src/components/dynamic/MapComponent.tsx:23]: Can we put this into a single utility function and re-use it across the project?

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by jonkoops** [comment]
Perhaps a better solution for generating IDs in React components would be to use the [`useId()`](https://beta.reactjs.org/reference/react/useId) hook which was introduced in React 18.

**Review by jonkoops — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**